corehq/apps/dashboard/views.py

```diff
@@ -177,7 +177,7 @@ def _can_access_reminders(request):
         and request.couch_user.can_edit_data()
     )
 
-    is_domain_admin = lambda request: request.couch_user.is_domain_admin(request.domain)
+    is_billing_admin = lambda request: request.couch_user.has_permission(request.domain, 'billing_admin')
```

Comment:
yeah, `has_permission` just always returns true for domain admins, it doesnt look at the specific one. as far as constants, `billing_admin` is really an attribute of a permission object, so its not stored in a string constant anywhere. the call there is equivalent to the automagically generated function `can_billing_admin()` but the weird grammar there made this feel more reasonable.